src/CSET/operators/plot.py

```diff
@@ -643,7 +661,9 @@ def plot_vertical_line_series(
         Name of the plot to write, used as a prefix for plot sequences. Defaults
         to the recipe name.
     series_coordinate: str, optional
-        Coordinate to plot on the y-axis. Defaults to ``pressure``.
+        Coordinate to plot on the y-axis. Can be model_level for um or
+        full_levels for lfric, but defaults to ``model_level_number`` as lfric data
+        do not contain a pressure coordinate.
```

Comment:
```suggestion
        Coordinate to plot on the y-axis. Can be ``pressure`` or
        ``model_level_number`` for UM, or ``full_levels`` or ``half_levels``
        for LFRic. Defaults to ``model_level_number``.
```
If you accept the above change this should be updated to specify `pressure` as the default.

Reply:
as not accepting the change to pressure I won't commit this.

Reply:
The change is valid for model_level_number currently. The suggestion would have had to have been changed if we went for ``pressure`` as default, but as we have not it can be committed.